src/app/app.module.ts

```diff
@@ -0,0 +1,49 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+
+// import { FormsModule } from '@angular/forms';
+// import { ReactiveFormsModule } from '@angular/forms';
```

Comment:
Why you need this? Remove if it is unused, or at least add a comment on why you leave this.